### ISSUE TITLE
MiKo_1086 reports wrong slang

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#369]
```c#
[Test]
public void Foo_the_2nd_Bar_the_4th() // MiKo_1086 Do not use a number as slang
{
    Assert.Fail();
}
```

Maybe tests should get ignored by that rule.
